esmf_regrid/tests/unit/experimental/unstructured_scheme/test_MeshToGridESMFRegridder.py

```diff
@@ -346,3 +347,53 @@ def test_resolution():
     lat_band_rg = MeshToGridESMFRegridder(mesh_cube, lat_bands, resolution=resolution)
     assert lat_band_rg.resolution == resolution
     assert lat_band_rg.regridder.tgt.resolution == resolution
+
+
+def test_curvilinear():
+    """
+    Test for :func:`esmf_regrid.experimental.unstructured_scheme.MeshToGridESMFRegridder`.
```

Comment:
This feels like it builds a lot of similar stuff to the test for the other direction - can we share the code that builds the cubes in helper functions?